cfme/tests/cloud/test_cloud_manual.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream("5.9", "5.10", "upstream")]
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@test_requirements.discovery
+def test_add_cloud_provider_screen():
```

Comment:
It looks like a lot of this is covered by automated tests in cfme/tests/cloud/test_providers.py.

Few comments/things to do:

1: Please re-format this docblock to remove any steps that are covered by automation already, and put any remaining test steps in the Polarion docblock like:
```
Polarion:
    testSteps:
        1. test cloud provider add with invalid hostname syntax
        2. test cloud provider with invalid security protocol
```

2. This looks like it should be parametrized by provider with a common set of steps to test.

3. Consider removing the method completely by automating whatever add form fields aren't already tested.